config/services.yml

```diff
@@ -23,6 +23,35 @@ x-turbo-config: &default-turbo-config
 
 https://cmr.earthdata.nasa.gov:
 
+  - name: asf/opera-rtc-s1-browse
```

Comment:
We'll want to create another copy of this service under https://cmr.uat.earthdata.nasa.gov before merging into nasa/harmony.